pySPT/notebookspy/trackAnalysis_noGUI.py

```diff
@@ -0,0 +1,88 @@
+"""
+@author: Alexander Niedrig, Johanna Rahm, Claudia Catapano
+Research group Heilemann
+Institute for Physical and Theoretical Chemistry, Goethe University Frankfurt a.M.
+
+virtual python only version of the TrackAnalysis.ipynb notebook for batch processing analysis
+"""
+from pySPT.Analysis import coverSlip
+from pySPT.Analysis import trackAnalysis
+from pySPT.widgets import widgetColumnSort
+from pySPT.widgets import widgetDirectoryStructure
+from pySPT.widgets import widgetTrackAnalysis
+from pySPT.widgets.widgetNotebooks import init_save_track_analysis
+import time
+from tqdm import *
```

Comment:
importing with * gives an unknown namespace. Please import only the necessary modules.